api/src/test/java/org/pmiops/workbench/api/UserMetricsControllerTest.java

```diff
@@ -152,19 +171,21 @@ public void setUp() {
 
     userMetricsController =
         new UserMetricsController(
-            userProvider,
-            userRecentResourceService,
-            workspaceService,
-            fireCloudService,
-            cloudStorageService,
-            clock);
+            mockUserProvider,
+            mockUserRecentResourceService,
+            mockWorkspaceService,
+            mockFireCloudService,
+            mockCloudStorageService,
+            fakeClock);
     userMetricsController.setDistinctWorkspaceLimit(5);
+
+    cohortMapper = new CohortMapperImpl();
```

Comment:
this line should be unnecessary with autowired

Reply:
oh right, thanks.